src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1534,6 +1576,13 @@ internal StyleViewModel(OptionStore optionStore, IServiceProvider serviceProvide
                 new CodeStylePreference(CSharpVSResources.Prefer_explicit_type, isChecked: false),
             };
 
+            var usingDirectivesPlacementPreferences = new List<CodeStylePreference>
+            {
+                new CodeStylePreference(CSharpVSResources.Preserve, isChecked: false),
+                new CodeStylePreference(CSharpVSResources.Inside_namespace, isChecked: false),
+                new CodeStylePreference(CSharpVSResources.Outside_namespace, isChecked: false),
+            };
```

Comment:
having a 'preserve' option seems odd to me.  we don't generlaly have that.  our position up till now has been that we always have a preference, but we can have no-severity, meaning we won't tell you there's an issue, or otherwise try to get you to change it.

Reply:
Sam brought up that when we update Organize Imports it would use the Preserve option to mean don't move imports. There is also the Add Import code fix, if we didn't have the Preserve option then the current location of using directives would get ignored in favor of the code style setting we defaulted to.

Reply:
> here is also the Add Import code fix, if we didn't have the Preserve option then the current location of using directives would get ignored in favor of the code style setting we defaulted to.


Hrmm... for add-import, i think we should respect the file no matter what the option is.  Thoughts?